### ISSUE TITLE
Update WLED to Aid Functionality

### ISSUE THREAD

**Issue by Stewie-040882** [s-marley/Infinity-Cube#1]
I dont know if youre aware, but using WLED you can enter the lights section, and using only 1 pin, you can break 1 long led strip into multiple segments, and control each segment independently, by dictating how many leds make 1 segment..
   
Also, as a side note, i would make the base plate, the housing for the esp32. So you would have the display pedestal (containing the electronics), and the cube. That way it can be admired on a shelving, or taken off to admire it..

Great work by the way, i thought the idea was amazing!!

**Comment by s-marley** [comment]
Yeah absolutely. I have used WLED for [other projects](https://www.youtube.com/watch?v=9iJQhOiL2bk&t=2s), I just quite enjoy writing my own patterns and raw FastLED is much quicker to write and iterate on than recompiling WLED with a custon FX.cpp each time. I'm trying to remember why I didn't build the controller into the base, there was a reason back then but it escapes me now! Thank you for your comment :)

**Comment by Stewie-040882** [comment]
Cool, sorry, just when you mentioned WLED, but ive just rewatched the section referencing WLED.. shame really about the 10 segments, but i know those Neopixel leds use the same logic (1 continuous strip), and was figuring it would save time coding (as i can only read code lol).

i was picturing the cube with 12 strips showing the 3d fireworks. 

I know you mentioned you like your own code, which i respect, someone with coding knowledge could be able to modify their opensource code. or request more segments on their github.

apologies as im sure youve considered this option. Im still going to attempt a build of your project at some point, but think ill try with the larger leds first as i know it will be extremely tricky to get it all to fit

**Comment by s-marley** [comment]
No problem, just doing what I think is fun :) and yes, if you're going to make your own, I'd definitely start with a bigger version, those tiny LEDs were a right pain even for someone who has a bit of experience in this area. Good luck!
